src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+header {
```

Comment:
Consider using a more specific selector for the header, such as '.header' instead of the generic 'header' tag. This will help avoid conflicts with other potential header elements on the page.